src/data_processing.rs

```diff
@@ -20,7 +25,9 @@ pub fn generate_world(
 
     let region_template_path: &str = "region.template";
     let region_dir: String = format!("{}/region", args.path);
-    let ground_level: i32 = -62;
+
+    let ground = Ground::new();
+    let ground_level = 60; // TODO
```

Comment:
You'll see these in a few spots. They're only used by elements that I haven't updated to use the `Ground` struct yet.